providers/tentacle.rb

```diff
@@ -71,16 +72,30 @@
     version version
   end
 
-  configure = batch "configure-tentacle-#{instance}" do
+  generate_cert = powershell_script 'generate-tentacle-cert' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.